docs/source/experiments/regression-robust04.md

```diff
@@ -0,0 +1,80 @@
+# PISA: Regressions for [Disks 4 & 5](https://trec.nist.gov/data_disks.html) (Robust04)
+
+## Indexing
+
+First, we will create a directory where all the indexes are going to be stored:
+
+```
+mkdir robust04
+```
+
+### Parsing
+
+```
+gzip -dc $(find /path/to/disk45/ -type f -name '*.*z' \( -path '*/disk4/fr94/[0-9]*/*' -o -path '*/disk4/ft/ft*' -o -path '*/disk5/fbis/fb*' -o -path '*/disk5/latimes/la*' \)) | bin/parse_collection -f trectext -b 10000 --stemmer porter2 --content-parser html -o robust04/fwd
+```
+
+You can replace `gzip -dc` with `zcat` on Linux or `gzcat` on MacOS.
+The directory `/path/to/disk45/` should be the root directory of [TREC Disks 4 & 5](https://trec.nist.gov/data_disks.html).
+
+### Inverting
+
+```
+bin/invert -i robust04/fwd -o robust04/inv -b 400000 --term-count $(cat robust04/fwd.terms | wc -l)
```

Comment:
I think we can ignore the `bin` part, especially since we're not even saying how we build it and where we're running it from, don't you?

Reply:
Yes agree, could just do `/path/to/pisa/build/bin` or something instead